### DISCUSSION TITLE
How can I show message in pull request merge box for finished action/workflow?

### DISCUSSION THREAD

**Discussion by xMort** [community/community#37469] [Actions]
I have an action that does some soft check on commit messages. When action/workflow finishes with warning, I want to show simple text to the user or at least change the check icon to a warning. Is it possible to do it via GitHub action? I only find out about annotations put on the specific files.

I was thinking about posting a comment to pull request but I would have to update GITHUB_TOKEN with `issues: write` permissions as these are pull requests from private fork to protected master branch.

Displaying a line of text below the check or changing its status would be the best solution. The only thing I found is a possibility to fail the job that that is too harsh for my use case.

<img width="920" alt="image" src="https://user-images.githubusercontent.com/1880148/198235494-22125f57-0707-4c7c-97a8-3d2bace8a975.png">

**Comment by ossanna16** [comment]
Hi there @xMort and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
